Week3/homework/fortuneTeller.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+{
+    function tellFortune(){
+        
+        return "You will be a " + jobs[Math.floor(Math.random() * jobs.length)] + " in " + locations[Math.floor(Math.random() * locations.length)] + ", and married to " + partnerNames[Math.floor(Math.random() * partnerNames.length)] + " with " + numChildren[Math.floor(Math.random() * numChildren.length)] + " kids."
+    }
+
+    let partnerNames = ["Ayse", "Fatma", "Hayriye", "Kezban", "Hatice"];
+    let numChildren = [1, 2, 3, 4, 5];
+    let locations = ["Amsterdam", "Rotterdam", "Alkmar", "Hengelo", "Tilburg"];
+    let jobs = ["programmer", "doctor", "teacher", "lawyer", "officer"];
+
+    console.log(tellFortune());
```

Comment:
Ah. Couple things here:
1. You can explore a new feature in JavaScript called template literals(https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals) after which you need not use + operator to concatenate strings
2. You were required to pass all variables as function arguments instead of using them in the function directly. 🙂